src/js/letters/actions/letters.js

```diff
@@ -123,19 +142,35 @@ export function getLetterPdf(letterType, letterName, letterOptions) {
             }
           }
         });
-        window.URL.revokeObjectURL(downloadUrl); // make sure this doesn't cause problems
+        window.URL.revokeObjectURL(downloadUrl);
         dispatch({ type: GET_LETTER_PDF_SUCCESS, data: letterType });
       },
       () => dispatch({ type: GET_LETTER_PDF_FAILURE, data: letterType })
     );
   };
 }
 
-
 export function updateBenefitSummaryRequestOption(propertyPath, value) {
   return {
     type: UPDATE_BENFIT_SUMMARY_REQUEST_OPTION,
     propertyPath,
     value
   };
 }
+
+export function updateAddress() {
+  const settings = {
+    method: 'PATCH',  // TODO: decide whether to use PATCH or PUT here; check with Alastair
```

Comment:
@kreek can we arbitrarily choose to PATCH or PUT here? Since we have to provide a valid complete address in the request body, I'm guessing a PUT makes more sense.

Reply:
@raquelromano-gov yes PUT since even with partial updates we send the complete address.